src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] data = line.split(",");
+                if (data.length == 2) {
+                    String operationType = data[0].trim();
+                    int amount = Integer.parseInt(data[1].trim());
+
+                    if ("supply".equalsIgnoreCase(operationType)) {
+                        supply += amount;
+                    } else if ("buy".equalsIgnoreCase(operationType)) {
+                        buy += amount;
+                    }
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Using `e.printStackTrace()` is not suitable for production code. Consider re-throwing a `RuntimeException` with a meaningful message and the original exception to help in better error tracking and debugging.